app/index.js

```diff
@@ -1,5 +1,7 @@
 const electron = require("electron");
 const path = require("path");
+const ipc = require("electron").ipcMain;
```

Comment:
why not just `electron.ipcMain`? why re-require, in most implementations the require will be cached so it doesn't matter but either way